pt-BR/mail/chrome/messenger/dataPane.dtd

```diff
@@ -1,77 +1,77 @@
+<!ENTITY dataPaneContext.detachAttachment.label       "Retirar...">
+<!ENTITY dataPaneContext.deleteAttachment.label       "Excluir">
+<!ENTITY dataPaneContext.saveAll.label                "Salvar todos...">
 <!ENTITY dataPaneContext.saveAll.accesskey           "v">
-<!ENTITY dataPaneContext.systemPhotoImportMac.label  "Add to iPhoto">
+<!ENTITY dataPaneContext.systemPhotoImportMac.label  "Adicionar ao iPhoto">
 <!ENTITY dataPaneContext.systemPhotoImportMac.accesskey  "i">
 
 <!-- contact panel -->
-<!ENTITY contactPanel-name.label  "name:">
-<!ENTITY contactPanel-title.label "title:">
-<!ENTITY contactPanel-company.label  "company:">
+<!ENTITY contactPanel-name.label  "nome:">
+<!ENTITY contactPanel-title.label "título:">
+<!ENTITY contactPanel-company.label  "empresa:">
 
-<!ENTITY contactPanel-phone.label "phone:">
-<!ENTITY contactPanel-work.label "(work)">
-<!ENTITY contactPanel-home.label "(home)">
-<!ENTITY contactPanel-mobile.label "(mobile)">
+<!ENTITY contactPanel-phone.label "telefone:">
+<!ENTITY contactPanel-work.label "(empresa)">
+<!ENTITY contactPanel-home.label "(residencial)">
+<!ENTITY contactPanel-mobile.label "(celular)">
 
 <!ENTITY contactPanel-email.label "email:">
-<!ENTITY contactPanel-homeAddress.label "home:">
-<!ENTITY contactPanel-workAddress.label "work:">
+<!ENTITY contactPanel-homeAddress.label "residencial:">
+<!ENTITY contactPanel-workAddress.label "trabalho:">
 <!ENTITY contactPanel-onlineStatus.label "IM:">
 <!ENTITY contactPanel-facebook.label     "Facebook:">
 <!ENTITY contactPanel-twitter.label      "twitter:">
 
-<!ENTITY contactPanel-favorite.label     "favorite:">
-<!ENTITY contactPanel-favorite-checkbox.label "make this contact a favorite">
-<!ENTITY contactPanel-actions.label "actions:">
-<!ENTITY contactPanel-find.label    "find recent:">
-<!ENTITY contactPanel-findMessages.label "messages">
-<!ENTITY contactPanel-findDocuments.label "attachments">
-<!ENTITY contactPanel-findImages.label "images">
-<!ENTITY contactPanel-copyContact.label "copy email">
-<!ENTITY contactPanel-sendContact.label "send vcard">
-<!ENTITY contactPanel-createFilter.label "create filter...">
+<!ENTITY contactPanel-favorite.label     "favoritos:">
+<!ENTITY contactPanel-favorite-checkbox.label "colocar este contato como favorito">
+<!ENTITY contactPanel-actions.label "ações:">
+<!ENTITY contactPanel-find.label    "encontrar recentes:">
+<!ENTITY contactPanel-findMessages.label "mensagens">
+<!ENTITY contactPanel-findDocuments.label "anexos">
+<!ENTITY contactPanel-findImages.label "imagens">
+<!ENTITY contactPanel-copyContact.label "copiar email">
+<!ENTITY contactPanel-sendContact.label "enviar vcard">
+<!ENTITY contactPanel-createFilter.label "criar filtro...">
 
 <!-- action contact panel -->
-<!ENTITY contactActionPanel-sendMail.label "Send Mail...">
-<!ENTITY contactActionPanel-copyAddress.label "Copy Email Address">
+<!ENTITY contactActionPanel-sendMail.label "Enviar email...">
```

Comment:
CORRECTIONS NEEDED it should be <!ENTITY contactActionPanel-sendMail.label "Enviar mensagem...">